### PR TITLE
Make Chinese (Traditional) translation better

### PR DESCRIPTION
**Pull request by pan93412**
Also update some new translations :)

### CODE REVIEW

**Comment by pan93412** [comment]
> Looks good to me

:) thanks

File: src/main/resources/lang/installer_zh_TW.properties [hunk 1]
```diff
@@ -1,24 +1,37 @@
...
+progress.done.server=伺服器安裝成功
+progress.download.libraries=正在下載必要檔案…
+progress.download.minecraft=正在下載 Minecraft 伺服器
+progress.download.library.entry=正在下載 {0} 函式庫
+progress.exception.no.launcher.directory=未找到啟動器目錄！
+progress.generating.launch.jar=正在產生伺服器啟動 JAR
+progress.generating.launch.jar.library=正在產生伺服器啟動 JAR：{0}
+progress.installing=正在客戶端安裝 {0} Fabric 啟動器
+progress.installing.server=正在伺服器安裝 {0} Fabric 啟動器
+prompt.exception=例外
```
Comment (shedaniel) [inline src/main/resources/lang/installer_zh_TW.properties]: Oops missed this, this means an error not literally an exception.

Comment (pan93412) [reply src/main/resources/lang/installer_zh_TW.properties]: [Chinese] 實際上那並不是「錯誤」，他的實際意思是：**原本我預期流程應該是這樣走，但是中間突然發生了例外狀況**，而不是普通的錯誤，他更傾向於「原本沒預期到的錯誤」，是「錯誤」的子集。個人認為使用「例外」會比較好。

附上一些參考資料： [Yahoo 字典](https://tw.dictionary.search.yahoo.com/search?p=exception)、[Microsoft 詞彙庫](https://www.microsoft.com/zh-tw/language/Search?&searchTerm=except&langID=129&Source=true&productid=0)

File: src/main/resources/lang/installer_zh_TW.properties [hunk 2]
```diff
@@ -1,24 +1,37 @@
...
+prompt.exception.caused.by=發生原因：
+prompt.exception.occurrence=遇到例外事件！
+prompt.exception.suppressed=已隱藏
 prompt.install=安裝
-prompt.loader.version=加載器版本：
-prompt.loading.versions=正在加載可安裝版本……
-prompt.game.version=遊戲和反混淆數據版本：
-prompt.ready.install=已準備完畢，可以安裝
+prompt.loader.version=啟動器版本：
```
Comment (liach) [inline src/main/resources/lang/installer_zh_TW.properties]: this is not a launcher but a mod loader to be exact. more like 读取器 or what?

File: src/main/resources/lang/installer_zh_TW.properties (hunk 1 above)
Comment (pan93412) [reply src/main/resources/lang/installer_zh_TW.properties]: Anyway 我自己更傾向於使用詞彙原有的意思：「Exception」，不然就把上游的字串直接改成「Error」就好啦 :)

Comment (liach) [reply src/main/resources/lang/installer_zh_TW.properties]: In mainland china this is called "异常". Try use java programmer terms for `java.lang.Exception`.

File: src/main/resources/lang/installer_zh_TW.properties [hunk 3]
```diff
@@ -1,24 +1,37 @@
...
+prompt.loading.versions=正在載入版本
+prompt.game.version=Minecraft 版本：
+prompt.ready.install=安裝準備就緒
 prompt.select.location=選擇安裝位置
+prompt.server.info.jar=執行 Fabric 需要官方的 Minecraft 伺服器 jar
+prompt.server.info.command=使用這個指令來啟動伺服器
+prompt.server.info.scipt=或產生啟動文稿
```
Comment (liach) [inline src/main/resources/lang/installer_zh_TW.properties]: feels like 脚本, like a shell/windows bat script

File: src/main/resources/lang/installer_zh_TW.properties (hunk 2 above)
Comment (pan93412) [reply src/main/resources/lang/installer_zh_TW.properties]: Good suggest! :)

File: src/main/resources/lang/installer_zh_TW.properties (hunk 3 above)
Comment (pan93412) [reply src/main/resources/lang/installer_zh_TW.properties]: Exception: "異常" in traditional chinese is: "例外".
Script: 「文稿」就是「腳本」的意思，也可以翻成「指令稿」「指令碼」。

File: src/main/resources/lang/installer_zh_TW.properties (hunk 2 above)
Comment (shedaniel) [reply src/main/resources/lang/installer_zh_TW.properties]: `加載器`?

Comment (pan93412) [reply src/main/resources/lang/installer_zh_TW.properties]: @shedaniel 我通常都使用台灣用詞翻譯，那應該是中國用詞。

File: src/main/resources/lang/installer_zh_TW.properties (hunk 1 above)
Comment (shedaniel) [reply src/main/resources/lang/installer_zh_TW.properties]: `例外錯誤` should work.

**Comment by pan93412** [comment]
Ok, looks resolved.

File: src/main/java/net/fabricmc/installer/util/CrashDialog.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+import javax.swing.*;
+import java.awt.*;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+public class CrashDialog {
+
+	public CrashDialog(Throwable throwable) {
+		JFrame frame = new JFrame("Fabric installer has crashed!");
```
Comment (liach) [inline src/main/java/net/fabricmc/installer/util/CrashDialog.java]: consider translating this with the [util bundle](https://github.com/FabricMC/fabric-installer/blob/cfe5a39132d5ff4ceb4b3421229f5c54bbc18134/src/main/java/net/fabricmc/installer/util/Utils.java#L36)?

File: src/main/resources/lang/installer_zh_TW.properties (hunk 2 above)
Comment (shedaniel) [reply src/main/resources/lang/installer_zh_TW.properties]: `載入程式` seems reasonable, but why not `載入器`? Sorry I am not familiar with neither the Taiwan way or the China way.

File: src/main/java/net/fabricmc/installer/util/CrashDialog.java (hunk 4 above)
Comment (pan93412) [reply src/main/java/net/fabricmc/installer/util/CrashDialog.java]: @modmuss50

Comment (modmuss50) [reply src/main/java/net/fabricmc/installer/util/CrashDialog.java]: No, I think keeping this basic crash dialong free of as much code behind it is best. Idealy people will never see it

**Comment by modmuss50** [comment]
Something seems up with the PR as its including changes that I pushed to master.

**Comment by pan93412** [comment]
yeah, just ignore it?

**Comment by pan93412** [comment]
@modmuss50 So, how about this Pull Request?

File: src/main/java/net/fabricmc/installer/Main.java [hunk 5]
```diff
@@ -67,7 +64,12 @@ public static void main(String[] args) throws IOException, ClassNotFoundExceptio
 		setDebugLevel(Level.SEVERE);
 
 		if (command == null) {
-			InstallerGui.start();
+			try {
```
Comment (modmuss50) [inline src/main/java/net/fabricmc/installer/Main.java]: This change shouldnt be here

**Comment by pan93412** [comment]
@modmuss50 Ok, I've resolved that issue.

**Pull request merged**